librarian_resources/Manipulate request
---
bool status
